src/components/NewsDemo.tsx

```diff
@@ -0,0 +1,34 @@
+import { NewsProps } from "@/lib/types";
+import { fetchData } from "@/lib/api";
+
+export const NewsDemo = async () => {
+  const response = await fetchData<{ data: NewsProps[] }>("items/news");
+  const news = response.data.map((post) => ({
+    ...post,
+    date_created: new Date(post.date_created),
+  }));
+
+  // Sortowanie news'ów od najnowszych, w przyszłości może to nie wystarczyć (przypięte/customowa kolejność)
+  const newsSortedByDate = news.sort(
+    (a, b) => b.date_created.getTime() - a.date_created.getTime()
+  );
```

Comment:
Poki co wystarczy odwrocic liste, bo dane sa autonumerowanych indexach, przez zrobi sie liniowa zlozonosc
```suggestion
  const newsSortedByDate = news.reverse();
```